src/main.js

```diff
@@ -50,6 +51,7 @@ class CustomElementsLocator extends LitElement {
       ${this.loaded
         ? this.customElements.length > 0
           ? html`
+              <h2>I found custom elements on </h2>
```

Comment:
I think we can remove this line

Reply:
I considered it this afternoon. But without that line, will it be obvious enough?

Reply:
Done